.eslintrc.json

```diff
@@ -1,17 +1,21 @@
 {
   "extends": [
-    "okonet"
+    "okonet",
+    "plugin:node/recommended"
```

Comment:
Just FYI the latest version of my config has node plugin integrated so we might want to replace it with it. But I'm fine with this as well. It's more explicit but will probably make it harder for me to maintain across projects.

Reply:
great! i'll update our ESLint and also use your the latest version of `eslint-config-okonet`

Reply:
but your `postinstall` script doesn't work. we still need to manually install the peerdeps manually.

maintaining or bundling shareable config is painful IMO. I had mine but then abandoned it, I'm using XO now

Reply:
![screen shot 2017-09-16 at 16 42 10](https://user-images.githubusercontent.com/22868432/30511209-29a5c2f0-9afe-11e7-803d-e38a00cf2105.png)

Reply:
![screen shot 2017-09-16 at 16 48 51](https://user-images.githubusercontent.com/22868432/30511256-f735d444-9afe-11e7-8356-ae9705683dc1.png)

Reply:
Yeah, I'm aware of that. That's why I wasn't suggesting upgrading to it yet.

Reply:
Should we go XO as well?

Reply:
> Should we go XO as well?

Based on my experience, XO is great! I'm using it in all of my projects
But we don't have to use it if you feel uncomfortable

These are some options:
1. Use `eslint-plugin-airbnb-base` and all of its peerDeps (tweak the config as needed)
2. `eslint-config-okonet` still can be used, but I suggest to expose the shareable config only, leave peerDeps to be installed manually